python-avd/pyavd/_eos_designs/schema/schema_fragments/dot1x_settings.schema.yml

```
@@ -0,0 +1,70 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  dot1x_settings:
+    documentation_options:
+      table: dot1x-settings
+    type: dict
+    description: Settings for 802.1X deployments.
+    keys:
+      enabled:
+        type: bool
+        default: false
+        description: |-
+          Globally enable 802.1X port authentication on the switch.
+          Must be set for 802.1X to be active on any interface.
+          When set, `dot1x_settings.radius_groups` is required.
+      radius_groups:
+        type: list
+        description: |-
+          List of RADIUS server groups to be used for 802.1X authentication and accounting.
+          Automatically generates `aaa_settings.authentication.dot1x.default` and `aaa_settings.accounting.dot1x.default`, overriding any manual settings.
```

Instead of overriding, we could instead let `radius_groups` be optional and honor user provided settings.
1- `radius_groups` defined --> we override any user provided settings
2- `radius_groups` undefined --> we honor user settings

For 2, we could raise if not provided or if any group is not defined under `aaa_settings.radius.servers`.

---

None of this seems very intuitive. We could deprecate setting dot1x under aaa_settings and move it here?

---

Removed the `dot1x` keys from `aaa_settings` and updated `dot1x_settings` schema to support them.